src/sections/create-dataset/AuthorFormGroup.tsx

```diff
@@ -0,0 +1,78 @@
+import { Form } from '@iqss/dataverse-design-system'
+import { useTranslation } from 'react-i18next'
+import { SubmissionStatus } from './useCreateDatasetForm'
+import { DatasetMetadataSubField } from '../../dataset/domain/models/Dataset'
+import { Col, Row } from '@iqss/dataverse-design-system'
+import { DynamicFieldsButtons } from './dynamic-fields-buttons/DynamicFieldsButtons'
+
+import { ChangeEvent } from 'react'
+import _ from 'lodash'
+import { useMultipleFields } from './useMultipleFields'
+interface AuthorFormGroupProps {
+  submissionStatus: SubmissionStatus
+  initialAuthorFields: DatasetMetadataSubField[]
+  updateFormData: (name: string, value: string | DatasetMetadataSubField[]) => void
+}
+
+export function AuthorFormGroup({
```

Comment:
done!  I noticed that findByLabelText() wasn't working, I think because the "Author Name" wasn't attached to the input.  So I created another label, 'Name' that is grouped with the input field.  This also matches what we want for compound fields.